tests/widget/inline/widget-inline.es.test.js

```diff
@@ -1,14 +1,19 @@
 import { expect, test } from '@playwright/test';
 
-test('inline widget in Spanish with unknown country', async ({ browser }) => {
-  const context = await browser.newContext({ locale: 'es' });
-  const page = await context.newPage();
-
+test.use({ locale: 'es' });
+test('inline widget in Spanish with unknown country', async ({ page }) => {
   await page.goto('widget/inline');
 
-  await page.fill('text="Nombre de usuario"', 'demouser');
-  await page.fill('text=Contraseña', 'j56eKtae*1');
-  await page.locator('button', { hasText: 'Iniciar sesion' }).click();
+  const username = page.getByRole('textbox', { name: "Nombre de usuario" });
+
+  await username.fill('demouser')
+
+  const password = page.getByRole('textbox', { name: "Contraseña" });
+  await password.fill('j56eKtae*1');
+
+  const submit = await page.getByRole('button', { name: 'Iniciar sesion' })
```

Comment:
I do like how this aligns the convention with Jest/Testing Library, but it is more verbose when we aren't actually using the variable afterwards. I see the `getBy*` convention better than `.locator`, just not sure it's a good replacement for when we just straight up want to trigger a user event, like `.fill`.